sdk/src/main/java/com/mapbox/maps/MapControllable.kt

```diff
@@ -73,4 +74,18 @@ interface MapControllable : MapboxLifecycleObserver {
    * Set [OnFpsChangedListener] to get map rendering FPS.
    */
   fun setOnFpsChangedListener(listener: OnFpsChangedListener)
+
+  /**
+   * Add [Widget] to the map.
+   */
+  @MapboxExperimental
+  fun addWidget(widget: Widget)
+
+  /**
+   * Remove [Widget] from the map.
+   *
+   * @return true if widget was removed
```

Comment:
does false mean that no such widget was present? could add couple of words about it here but that's a minor, up to u

Reply:
Kk